components/RequestModal/index.js

```diff
@@ -18,10 +18,12 @@ import { Textarea } from "@chakra-ui/textarea";
 import { useToast } from "@chakra-ui/toast";
 import { Switch } from "@chakra-ui/react";
 import { useForm } from "react-hook-form";
+import { useState } from "react";
```

Comment:
If we are not using useState anywhere we can get rid of it

Reply:
ah thanks!